packages/hooks/README.md

```diff
@@ -52,6 +52,12 @@ One notable difference between the JS and PHP hooks API is that in the JS versio
 -   `filters`
 -   `defaultHooks`
 
+
+#### Parameters
+
+- `hookName` unique name of the hook. It may contains only alphanumeric characters and dashes `[a-zA-z\-\_]`. Slashes are not permitted.
+- `namespace` namespace used by your application. Needed only when adding or removing an action or a filter.
```

Comment:
namespace is required, so I would change "Needed only..." to "Used..."

Reply:
"The namespace can only contain numbers, letters, dashes, periods, underscores and slashes." ... should take the    `vendor/plugin/function`. (from inline docs)